packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -113,7 +119,7 @@ abstract class TransitionRoute<T> extends OverlayRoute<T> {
 
   T _result;
 
-  void handleStatusChanged(AnimationStatus status) {
+  void _handleStatusChanged(AnimationStatus status) {
```

Comment:
I don't feel strongly either way. It doesn't seem to have any clients currently. If any of our customers use it, then we'll find out and can make it public again.